README.md

```diff
@@ -482,6 +482,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 
 ### Libraries
 
+- [Async::Ollama](https://github.com/socketry/async-ollama) - Streaming, asynchronous Ruby bindings for Ollama, supporting both chat and generate endpoints.
```

Comment:
Possible to add lower in the list – either a the bottom or next to other Ruby libraries? And then at some point in the future we'll re-organize this 😊

Reply:
Isn't the list alphabetical?

Reply:
https://github.com/ollama/ollama/pull/11544